Lib/typing.py

```diff
@@ -1736,7 +1744,7 @@ class Bar(Generic[*Ts]): ...
     Note that there is only some runtime checking of this operator. Not
     everything the runtime allows may be accepted by static type checkers.
 
-    For more information, see PEP 646.
+    For more information, see PEPs 646 and 692.
```

Comment:
PEP 692 is new in Python 3.12, so perhaps we should revert this change for the 3.11 backport:

```suggestion
    For more information, see PEP 646.
```